src/LowLevelMotionController.cpp

```diff
@@ -137,8 +131,27 @@ int main(int argc, char **argv)
     double hover_throttle;
     getPidParams(param_nh, throttle_pid, pitch_pid, roll_pid, yaw_pid, hover_throttle);
 
-    // Create a quad velocity controller. It will output angles corresponding to our desired velocity
-    QuadVelocityController quadController(throttle_pid, pitch_pid, roll_pid, yaw_pid, hover_throttle);
+    // Wait for a valid time in case we are using simulated time (not wall time)
+    while (ros::ok() && ros::Time::now() == ros::Time(0)) {
+        // wait
+        ros::spinOnce();
+    }
+
+    // Create a quad velocity controller. It will output angles corresponding
+    // to our desired velocity
+    double update_frequency;
+    nh.param("update_frequency", update_frequency, 60.0);
```

Comment:
I would prefer if this were in the other functions that get parameters to avoid breaking the convention. It's easier on future programmers if the settings are loaded in one spot.